force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -162,17 +223,21 @@ public with sharing class FlowControlLoopsCollections {
      */
     public static String repeatString(String inputString, Integer repeatCount) {
         // Initialize the result String
+        String result = '';
 
         // Use a for loop to append the inputString to the result string repeatCount times
-        /*
-        for () {
+        for (Integer x = 1; x <= repeatCount; x++) {
+            result += inputString;
 
             // If it is not the last iteration, add a semicolon to separate the strings
+            if (x < repeatCount) {
+                result += '; ';
+            }
+
```

Comment:
Good work. Take a look at the String methods repeat and removeEnd. 

https://developer.salesforce.com/docs/atlas.en-us.apexref.meta/apexref/apex_methods_system_string.htm-